src/Module.php

```diff
@@ -108,4 +118,9 @@ public function onAuthenticationPost(MvcAuthEvent $e)
 
         $this->container->setService('api-identity', $e->getIdentity());
     }
+
+    public function getMvcRouteListener()
```

Comment:
@weierophinney ok, how to check these listeners:
https://github.com/zfcampus/zf-mvc-auth/pull/120/files#diff-ab83393897d6271600e706b513fd1de0R73
?

I did something similar what you have done here:
https://github.com/zfcampus/zf-versioning/commit/a9013550326ea9df432ed1b606273411a554e9a8#diff-2cf9f9b61ffec8561183510e290d9bb0R68

I don't like this solution neither but couldn't come up with anything better.